instrumentation/trilogy/lib/opentelemetry/instrumentation/trilogy/instrumentation.rb

```diff
@@ -36,6 +55,24 @@ def require_dependencies
         def patch_client
           ::Trilogy.prepend(Patches::Client)
         end
+
+        def configure_propagator(config)
+          propagator = config[:propagator]
+          @propagator = case propagator
+          when 'vitess' then fetch_propagator(propagator, 'OpenTelemetry::Propagator::Vitess')
+          when 'none', nil then NoopPropagator.new
+          else
+            OpenTelemetry.logger.warn "The #{propagator} propagator is unknown and cannot be configured"
+            NoopPropagator.new
```

Comment:
I know the comment says api private and all, but should we not just use this NoopPropagator?

https://github.com/open-telemetry/opentelemetry-ruby/blob/04ddefc2d95865f94578b77b6d686cf1a984fea0/api/lib/opentelemetry/context/propagation/noop_text_map_propagator.rb#L11-L23

Similar to https://github.com/open-telemetry/opentelemetry-ruby/blob/04ddefc2d95865f94578b77b6d686cf1a984fea0/sdk/lib/opentelemetry/sdk/configurator.rb#L212

Reply:
The SDK configurator uses its own implementation here: https://github.com/open-telemetry/opentelemetry-ruby/blob/04ddefc2d95865f94578b77b6d686cf1a984fea0/sdk/lib/opentelemetry/sdk/configurator.rb#L13

It is cleanest to define our own. Since it isn't part of the public API, it could change without notice and break any consumers.